### PR TITLE
KEYCLOAK-14492 More polish translation and fixed language name translation

### PR DESCRIPTION
**Pull request by psztoch**

### CODE REVIEW

**Comment by VinodAnandan** [comment]
@psztoch Thank you for your contribution.

As mentioned in the following contribution guidelines, the commit message needs a JIRA number. Sometimes you may find the issue in JIRA backlog (https://issues.redhat.com/projects/KEYCLOAK/ ). If you can't find it, could you please create a JIRA issue and prefix it?

https://github.com/keycloak/keycloak/blob/master/CONTRIBUTING.md

**Comment by psztoch** [comment]
Sorry, but I can't add Jira issue.
RedHat SSO team can't solve problem with my account registration... and I can't login to your Jira system.

**Comment by VinodAnandan** [comment]
@psztoch I have created one - https://issues.redhat.com/browse/KEYCLOAK-14492 . I hope the team can fix the issues soon

File: themes/src/main/resources-community/theme/base/account/messages/messages_pl.properties [hunk 1]
```diff
@@ -1 +1,242 @@
...
+
+editAccountHtmlTitle=Edycja konta
+personalInfoHtmlTitle=Dane osobiste
+federatedIdentitiesHtmlTitle=Połączone tożsamości
+accountLogHtmlTitle=Dziennik konta
+changePasswordHtmlTitle=Zmień hasło
+deviceActivityHtmlTitle=Aktywność urządzeń
+sessionsHtmlTitle=Sesje
+accountManagementTitle=Zarządzanie kontem
+authenticatorTitle=Autentykator dwuetapowy
```
Comment (slaskawi) [inline themes/src/main/resources-community/theme/base/account/messages/messages_pl.properties:20]: Maybe better "Uwierzytelnienie dwuetapowe"?

File: themes/src/main/resources-community/theme/base/account/messages/messages_pl.properties [hunk 2]
```diff
@@ -1 +1,242 @@
...
+allFieldsRequired=Wszystkie pola są wymagane
+
+backToApplication=&laquo; Powrót do aplikacji
+backTo=Wróć do {0}
+
+date=Data
+event=Zdarzenie
+ip=IP
+client=Klient
+clients=Klienci
```
Comment (slaskawi) [inline themes/src/main/resources-community/theme/base/account/messages/messages_pl.properties:107]: @bdaw I always have a problem with this one: "Klienci" or "Klienty" ? Usually, I use "Aplikacje klienckie", so I don't hit this problem.

File: themes/src/main/resources-community/theme/base/account/messages/messages_pl.properties [hunk 3]
```diff
@@ -1 +1,242 @@
...
+client_admin-cli=Admin CLI
+client_realm-management=Zarządzanie strefą
+client_broker=Broker
+
+
+requiredFields=Wymagane pola
+allFieldsRequired=Wszystkie pola są wymagane
+
+backToApplication=&laquo; Powrót do aplikacji
+backTo=Wróć do {0}
```
Comment (slaskawi) [inline themes/src/main/resources-community/theme/base/account/messages/messages_pl.properties:101]: In the account console this gets rendered as "Wróć do Konsola administratora bezpieczeństwa". Perhaps we should put ":" at the end?

**Review by slaskawi — CHANGES_REQUESTED** [review]
Thanks for this contribution @psztoch !

Overall it looks great. I found a couple of small issues that could be fixed. Let's give @bdaw a few days to check it up. 

Also - may I ask you to squash all your commits?

**Comment by psztoch** [comment]
@slaskawi Fixed and squashed.

**Comment by psztoch** [comment]
I fixed some typos and add the reported to me demand for missing fields from OpenID connect.

**Comment by psztoch** [comment]
@bdaw Have you any progress in evaluation? ;-)

**Comment by psztoch** [comment]
What is problem with this pull request?!

**Comment by bdaw** [comment]
Looks good on my end. 

Sorry for the delay in the review. Vacation season and I missed it when catching up.

**Review by bdaw — DISMISSED** [review]

**Review by slaskawi — COMMENTED** [review]
@psztoch The code looks good. Could you please remove the merging commit from this PR? Once this is done, please ping me and I'll approve it.

**Comment by psztoch** [comment]
@slaskawi What is next?

**Comment by psztoch** [comment]
Panowie @slaskawi i @bdaw, czy mam rozumieć, że nie warto inwestować swojego zaangażowania w projekt Keycloak, bo jest to bezcelowe i tak nie zostanie wykorzystane?

**Review by slaskawi — APPROVED** [review]

**Comment by psztoch** [comment]
@bdaw ?!

**Comment by psztoch** [comment]
@slaskawi Czy mogę prosić Cię o wsparcie w zakresie tego pull request'a?

**Review by bdaw — APPROVED** [review]
lgtm

**Review by slaskawi — APPROVED** [review]

**Pull request merged**